src/widget/container/insets.rs

```diff
@@ -0,0 +1,74 @@
+use embedded_graphics::prelude::*;
+
+#[derive(Clone, Copy, Eq, PartialEq)]
+pub struct Insets {
+    pub left: u32,
+    pub right: u32,
+    pub top: u32,
+    pub bottom: u32,
+}
+
+impl Insets {
+    pub fn none() -> Self {
+        Self {
+            left: 0,
+            right: 0,
+            top: 0,
+            bottom: 0,
+        }
+    }
+
+    pub fn all(value: u32) -> Self {
+        Self {
+            left: value,
+            right: value,
+            top: value,
+            bottom: value,
+        }
+    }
+
+    pub fn horizontal(value: u32) -> Self {
+        Self {
+            left: value,
+            right: value,
+            top: 0,
+            bottom: 0,
+        }
+    }
+
+    pub fn vertical(value: u32) -> Self {
```

Comment:
Will probably end up being nice to have helpers for (just) `top()`/`bottom()`/`left()`/`right()`? And/or builders? (to avoid having to write out any `...: 0`'s)

Reply:
Yeah, that's a good idea. I'll leave it for now to decide later whether a builder/whatever else style would feel good in use.